Rakefile

```diff
@@ -36,4 +36,22 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task default: [:test, :rubocop]
+task default: [:test, 'rubocop:auto_correct']
```

Comment:
`rubocop:auto_correct` will change the code directly? Or we have a prompt before the changes? If we have a prompt, I am :+1: